spec/models/manageiq/providers/openstack/cloud_manager/vm_spec.rb

```diff
@@ -120,96 +120,96 @@
-        expect(vm.supports_disassociate_floating_ip?).to eq true
+        expect(vm.supports?(:disassociate_floating_ip)).to eq true
       end
 
-      it "checks disassociate_floating_ip is_available? when no floating ips are associated with the instance" do
+      it "checks disassociate_floating_ip is supported when no floating ips are associated with the instance" do
         expect(vm).to receive(:floating_ips).and_return([])
-        expect(vm.supports_disassociate_floating_ip?).to eq false
+        expect(vm.supports?(:disassociate_floating_ip)).to eq false
       end
     end
 
     context "snapshot actions" do
       it "supports snapshot_create" do
-        expect(vm.supports_snapshot_create?).to eq true
+        expect(vm.supports?(:snapshot_create)).to eq true
       end
 
       it "does not support snapshot_create on terminated VM" do
-        expect(terminated_vm.supports_snapshot_create?).to be_falsy
+        expect(terminated_vm.supports?(:snapshot_create)).to be_falsy
       end
 
-      it "checks remove_snapshot is_available? when snapshots are associated with the instance" do
+      it "checks remove_snapshot is supported when snapshots are associated with the instance" do
         expect(vm).to receive(:snapshots).and_return([1]) # fake a floating ip being associated
-        expect(vm.supports_remove_snapshot?).to eq true
+        expect(vm.supports?(:remove_snapshot)).to eq true
       end
 
-      it "checks remove_snapshot is_available? when no snapshots are associated with the instance" do
+      it "checks remove_snapshot is supported when no snapshots are associated with the instance" do
         expect(vm).to receive(:snapshots).and_return([])
-        expect(vm.supports_remove_snapshot?).to eq false
+        expect(vm.supports?(:remove_snapshot)).to eq false
       end
 
       it "does not support remove_snapshot_by_description" do
-        expect(vm.supports_remove_snapshot_by_description?).to eq false
+        expect(vm.supports?(:remove_snapshot_by_description)).to eq false
       end
 
       it "does not support revert_to_snapshot" do
-        expect(vm.supports_revert_to_snapshot?).to eq false
+        expect(vm.supports?(:revert_to_snapshot)).to eq false
       end
     end
 
     context "v2v actions" do
       it "supports conversion_host" do
-        expect(vm.supports_conversion_host?).to eq true
+        expect(vm.supports?(:conversion_host)).to eq true
       end
     end
   end
 
-  context "#is_available?" do
+  context "is supported?" do
     let(:ems) { FactoryBot.create(:ems_openstack) }
     let(:vm)  { FactoryBot.create(:vm_openstack, :ext_management_system => ems) }
     let(:power_state_on)        { "ACTIVE" }
     let(:power_state_suspended) { "SUSPENDED" }
 
     context("with :start") do
       let(:state) { :start }
-      include_examples "Vm operation is available when not powered on"
+      include_examples "Vm operation is supported when not powered on"
```

Comment:
@chessbyte `"Vm operation is available when not powered on"` is the name of a shared example from https://github.com/ManageIQ/manageiq/blob/master/spec/support/examples_group/shared_examples_for_vm_or_template_is_available.rb#L1

Reply:
Undid the shared example changes.